### PR TITLE
docker docker-completion 25.0.2

### PR DESCRIPTION
**Pull request by branchv**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/d/docker-completion.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@ class DockerCompletion < Formula
   desc "Bash, Zsh and Fish completion for Docker"
   homepage "https://www.docker.com/"
   url "https://github.com/docker/cli.git",
-      tag:      "v25.0.1",
+      tag:      "v25.0.2",
```
Comment (chenrui333) [inline Formula/d/docker-completion.rb:5]: it is the same revision but with diff tag?

Comment (branchvincent) [reply Formula/d/docker-completion.rb:5]: yea strange, might have been a mistake

Comment (p-linnane) [reply Formula/d/docker-completion.rb:5]: The revision for v25.0.1 is 293681613032e6d1a39cc88115847d3984195c24. Wonder if there was retagging. This should be fine to merge given that's the actual revision for v25.0.2.

**Review by chenrui333 — APPROVED** [review]

File: Formula/d/docker-completion.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/d/docker-completion.rb:5]: okay, merging.

**Pull request merged**